README.md

```diff
@@ -88,6 +88,7 @@ sudo passwd makerpm
 
 # install rpm build tools:
 sudo yum install rpm-build redhat-rpm-config rpmdevtools
+sudo yum groupinstall 'Development Tools'
```

Comment:
I'm afraid this group contains too many packages. Shall we just install the `gcc-c++` package?